pkg/crd/kubermatic/v1/user.go

```diff
@@ -52,7 +58,6 @@ type UserSpec struct {
 	IsAdmin                 bool                                    `json:"admin"`
 	Settings                *UserSettings                           `json:"settings,omitempty"`
 	TokenBlackListReference *providerconfig.GlobalSecretKeySelector `json:"tokenBlackListReference,omitempty"`
-	LastSeen                *metav1.Time                            `json:"lastSeen,omitempty"`
```

Comment:
not sure if we can just remove this as it will break compatibility. I think we need to deprecate it, and remove it in a future User version. 

Although this field is for internal use only and 99% nobody uses it, so maybe we can do a breaking change, but we would need to inform users in the release notes. 

BTW, as the sig-crd guys are working on CRD changes, we need to mirror this change in https://github.com/kubermatic/kubermatic/blob/master/pkg/apis/kubermatic/v1/user.go and update the migration in
https://github.com/kubermatic/kubermatic/blob/master/pkg/install/crdmigration/clones.go

Saying this, as we are anyway migrating the CRD, maybe the new one can have just the lastSeen in Status? @xrstf ?

Reply:
Keep in mind that the API version of this field hasn't changed. Only CRD will be different. I am fine with keeping it for now and deprecating until next release.

Reply:
Right, I misspoke, I meant the CRD compatibility will change. Although I think the chances are miniscule that somebody is integrating with our User CRD AND using the lastSeen, you never know. Plus we support integration through the API, not CRDs.

I would also be for removing it so we can forget about it, but would like some input from our SIG-CRD guys, anyway they need to approve this. 

Other then that, this PR LGTM

Reply:
> Saying this, as we are anyway migrating the CRD, maybe the new one can have just the lastSeen in Status? @xrstf ?

Your change in this PR looks totally fine to me, in fact I like moving this field into the `status` subresource. I don't quite understand the question, though ^^

Reply:
Sorry for the badly formulated question. 

Moving the field from `spec` to `status` is good for sure. The only concern is if we should remove the field from `spec` totaly, or should we leave a deprecated one there?

Reply:
Remove it :) Even if there was no migration, at worst we would lose this tiny piece of non-critical information.